ParalayoutTests/HairlineTests.swift

```diff
@@ -105,45 +110,63 @@ class HairlineTests: XCTestCase {
             Samples.window.screen = screen
-            
+
             Samples.hairline.isHorizontal = true
-            XCTAssert(Samples.hairline.sizeThatFits(CGSize(width: 100, height: 100)) == CGSize(width: 100, height: screen.hairlineWidth))
-            
+            XCTAssertEqual(Samples.hairline.sizeThatFits(CGSize(width: 100, height: 100)), CGSize(width: 100, height: screen.hairlineWidth))
+
             Samples.hairline.isHorizontal = false
-            XCTAssert(Samples.hairline.sizeThatFits(CGSize(width: 100, height: 100)) == CGSize(width: screen.hairlineWidth, height: 100))
+            XCTAssertEqual(Samples.hairline.sizeThatFits(CGSize(width: 100, height: 100)), CGSize(width: screen.hairlineWidth, height: 100))
         }
     }
-    
+
     func testSpanSuperview() {
+        // This test is broken in iOS 13 and later due to a bug around setting `UIWindow.screen` (FB8674601).
+        if #available(iOS 13, *) {
+            return
+        }
+
         // Left, 3x, inset, no mask.
         Samples.window.screen = TestScreen.at3x
         Samples.hairline.spanSuperview(at: .minXEdge, inset: 10, updateAutoresizingMask: false)
         XCTAssert(Samples.hairline.frame == CGRect(x: 0, y: 10, width: CGFloat(1) / 3, height: 30))
         XCTAssert(Samples.hairline.autoresizingMask == [])
         XCTAssert(!Samples.hairline.isHorizontal)
-        
+
         // Top, 2x, no inset, no resizing mask.
         Samples.window.screen = TestScreen.at2x
         Samples.hairline.spanSuperview(at: .minYEdge)
         XCTAssert(Samples.hairline.frame == CGRect(x: 0, y: 0, width: 100, height: 0.5))
         XCTAssert(Samples.hairline.autoresizingMask == [])
         XCTAssert(Samples.hairline.isHorizontal)
-        
+
         // Right, 2x, inset, with mask.
         Samples.window.screen = TestScreen.at2x
         Samples.hairline.spanSuperview(at: .maxXEdge, inset: -5, updateAutoresizingMask: true)
         XCTAssert(Samples.hairline.frame == CGRect(x: 99.5, y: -5, width: 0.5, height: 60))
         XCTAssert(Samples.hairline.autoresizingMask == [ .flexibleHeight, .flexibleLeftMargin ])
         XCTAssert(!Samples.hairline.isHorizontal)
-        
+
         // Bottom, 1x, no inset, with mask.
         Samples.window.screen = TestScreen.at3x
         Samples.hairline.spanSuperview(at: .maxYEdge, updateAutoresizingMask: true)
         XCTAssert(Samples.hairline.frame == CGRect(x: 0, y: 50 - UIScreen.hairlineWidth(for: 3), width: 100, height: UIScreen.hairlineWidth(for: 3)))
         XCTAssert(Samples.hairline.autoresizingMask == [ .flexibleWidth, .flexibleTopMargin ])
         XCTAssert(Samples.hairline.isHorizontal)
     }
+
+    // MARK: - Private Methods
+
+    private func screensToTest() -> [UIScreen] {
+        if #available(iOS 13, *) {
+            // In iOS 13 and later, there is a bug around setting `UIWindow.screen` that prevents us from testing
+            // multiple screens (FB8674601).
+            return [.main]
```

Comment:
Did we discover that adding an external display in the simulator wont work for us here?

Reply:
It _sort of_ works. We can get an external display with a scale of 1, but no customizable scale. The second screen also doesn't connect immediately when the simulator launches, so we'd need to work around that a bit in the tests.

For now, I think we should move forward with this approach and can look into the external displays again in the future if it becomes necessary.